module/documents/actor/actor.mjs

```diff
@@ -18,10 +18,17 @@ export default class Actor5e extends SystemDocumentMixin(Actor) {
 
   /**
    * The data source for Actor5e.classes allowing it to be lazily computed.
-   * @type {Object<Item5e>}
+   * @type {Record<string, Item5e>}
    * @private
    */
-  _classes;
+  #classes;
```

Comment:
A good idea, but because of some weirdness with how ActorDeltas are prepared, it seems to break preparation of unlinked token Actors.